static/usage/v6/modal/can-dismiss/prevent-swipe-to-close/angular/example_component_ts.md

```diff
@@ -0,0 +1,19 @@
+```ts
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent implements OnInit {
+  presentingElement = undefined;
+
+  ngOnInit() {
+    this.presentingElement = document.querySelector('.ion-page');
```

Comment:
Why not use `ViewChild`?

Reply:
`ViewChild` would be useful here if `.ion-page` was a child of the example component. We may be able to use `ElementRef` and find the closest element with `.ion-page` to treat as a presenting element. 

This example code was just copied from the existing callback example for `canDismiss`.